pipelines/build/common/build_base_file.groovy

```diff
@@ -654,12 +654,35 @@ class Builder implements Serializable {
             def downstreamJobName = 'build-scripts/release/package_binaries'
             context.echo "build name: ${downstreamJobName} for ${linuxTargets}"
 
+            def version = ''
+            def variantVersion = ''
+            def variantTags = ''
+
+            if (publishName && publishName.contains(variant)) {
```

Comment:
Can we get away with only the contains condition or will that error when it is null?

Reply:
No, it would throw a NPE when null.